lib/matplotlib/collections.py

```diff
@@ -853,36 +859,78 @@ def get_linewidth(self):
     def get_linestyle(self):
         return self._linestyles
 
+    def _set_mappable_flags(self):
+        """
+        Determine whether edges and/or faces are color-mapped.
+
+        This is a helper for update_scalarmappable.
+        It sets Boolean flags '_edge_is_mapped' and '_face_is_mapped'.
+
+        Returns
+        -------
+        mapping_change: bool
```

Comment:
Because it's private, it doesn't get rendered, but numpydoc requires this:
```suggestion
        mapping_change : bool
```

Reply:
Fixed.